src/hooks/useStrapi.ts

```diff
@@ -158,6 +161,17 @@ export const useStrapi = <T>({
     apiUrl.searchParams.set('filters[$or][1][campaignEnd][$null]', 'true');
   }
 
+  // partner-themes -->
+  if (contentType === 'partner-themes') {
```

Comment:
please us a const as for the rest of the if condition

Reply:
Nice, good find. Thanks!!